flink-runtime/src/test/java/org/apache/flink/runtime/testutils/CommonTestUtils.java

```diff
@@ -131,40 +128,16 @@ public static void printLog4jDebugConfig(File file) throws IOException {
         }
     }
 
-    public static void waitUntilCondition(
-            SupplierWithException<Boolean, Exception> condition, Deadline timeout)
-            throws Exception {
-        waitUntilCondition(condition, timeout, RETRY_INTERVAL);
-    }
-
-    public static void waitUntilCondition(
-            SupplierWithException<Boolean, Exception> condition,
-            Deadline timeout,
-            long retryIntervalMillis)
-            throws Exception {
-        waitUntilCondition(
-                condition, timeout, retryIntervalMillis, "Condition was not met in given timeout.");
-    }
-
-    public static void waitUntilCondition(
-            SupplierWithException<Boolean, Exception> condition, Deadline timeout, String errorMsg)
+    public static void waitUntilCondition(SupplierWithException<Boolean, Exception> condition)
```

Comment:
Are you referring to CommonTestUtils#waitUtil? Well I guess whoever added just didn't think things through. Nor checked for typos I'm afraid.

Reply:
Yes, I was wondering whether we should clean that up as part of this PR...

Reply:
That's gonna be annoying because the classes have the same name, so you may end up with fully-qualified imports within and current users of the runtime CommonTestUtils.